app/models/ability.rb

```diff
@@ -1,11 +0,0 @@
-class Ability
-  include CanCan::Ability
-
-  def initialize(user)
-    user ||= User.new
-
-    can [:read, :update], Dataset do |dataset|
-      user.creator_of_dataset?(dataset) || user.in_organisation?(dataset.organisation)
```

Comment:
I've asked @nicholsj for a steer on this. He said that we definitely need permissions for editing (and creating) datasets and that they should be tied to the owner organisation of the dataset.